actionview/lib/action_view/layouts.rb

```diff
@@ -405,11 +405,11 @@ def _normalize_layout(value)
     #
     # ==== Parameters
     # * <tt>formats</tt> - The formats accepted to this layout
-    # * <tt>require_layout</tt> - If set to true and layout is not found,
-    #   an +ArgumentError+ exception is raised (defaults to false)
+    # * <tt>require_layout</tt> - If set to +true+ and layout is not found,
+    #   an +ArgumentError+ exception is raised (defaults to +false+)
```

Comment:
Yup, found it described here: http://guides.rubyonrails.org/api_documentation_guidelines.html#booleans